### PR TITLE
[NEW] Add support to named colors, remove new lines on title from bots

### PR DESCRIPTION
**Pull request by rafaelks**
@RocketChat/ios

## Progress

- [x] Fixes
- [x] Tests

### CODE REVIEW

File: Rocket.Chat/Extensions/UIColorExtension.swift [hunk 1]
```diff
@@ -19,6 +19,24 @@ extension UIColor {
         )
     }
 
+    // MARK: Color from strings (good|warning|danger)
+
+    static func normalizeColorFromString(string: String) -> UIColor {
+        if string == "warning" {
```
Comment (cardoso) [inline Rocket.Chat/Extensions/UIColorExtension.swift]: What do you think of using a switch here instead?

File: Rocket.Chat/Extensions/StringExtensions.swift [hunk 2]
```diff
@@ -96,7 +96,11 @@ extension String {
     }
 
     func removingWhitespaces() -> String {
-        return components(separatedBy: .whitespaces).joined()
+        return components(separatedBy: .whitespacesAndNewlines).joined()
```
Comment (cardoso) [inline Rocket.Chat/Extensions/StringExtensions.swift]: Should we update the tests for this class later?

File: Rocket.Chat/Extensions/UIColorExtension.swift (hunk 1 above)
Comment (filipealva) [reply Rocket.Chat/Extensions/UIColorExtension.swift]: I think that using a switch would be better too. Also, what do you guys think about having a `NamedColor` enum holding the color names as member values and the colors itself as properties?

File: Rocket.Chat/Extensions/StringExtensions.swift [hunk 3]
```diff
@@ -96,7 +96,11 @@ extension String {
...
+    }
+
+    func removingNewLines() -> String {
+        return components(separatedBy: .newlines).joined()
```
Comment (cardoso) [inline Rocket.Chat/Extensions/StringExtensions.swift]: Can you add tests for this method in StringExtensionSpec.swift?

Example: string with spaces and newlines, but only newlines are removed

File: Rocket.Chat/Extensions/StringExtensions.swift (hunk 2 above)
Comment (rafaelks) [reply Rocket.Chat/Extensions/StringExtensions.swift]: ✅

File: Rocket.Chat/Extensions/StringExtensions.swift (hunk 3 above)
Comment (rafaelks) [reply Rocket.Chat/Extensions/StringExtensions.swift]: ✅

**Pull request merged**